### PR TITLE
Helm3 v2.5.0

### PR DESCRIPTION
**Pull request by DavidMarchant**
Add helm3 support for v2.5

Tested on Rancher

### CODE REVIEW

File: charts/storageos-operator/templates/storageoscluster_cr.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ apiVersion: storageos.com/v1
 kind: StorageOSCluster
 metadata:
   name: {{ .Values.cluster.name }}
-  namespace: {{ .Release.Namespace }}
-spec:
   namespace: {{ .Values.cluster.namespace }}
+spec:
```
Comment (mhmxs) [inline charts/storageos-operator/templates/storageoscluster_cr.yaml:8]: I have to add this line to be able to deploy the stuff:
```
secretRefNamespace: {{ .Values.cluster.namespace }}
```

**Review by mhmxs — APPROVED** [review]

**Comment by aeroniero33** [comment]
We need to bump the chart version for the CI to work

**Comment by aeroniero33** [comment]
Also the helm delete fails when the chart is installed in kube-system, only the operator gets deleted

**Review by mhmxs — APPROVED** [review]

**Review by angelos-p — APPROVED** [review]
LGTM

**Comment by aeroniero33** [comment]
We need to add automated tests for this however

File: charts/storageos-operator/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "v2.4.0"
+appVersion: "v2.5.0"
 description: Cloud Native storage for containers
 name: storageos-operator
 version: 0.4.7
```
[the remark this answers is not in the document]
Comment (DavidMarchant) [reply charts/storageos-operator/Chart.yaml]: Think you're looking at an old version there mate! Has been bumped to 0.5.0

**Pull request merged**